.github/workflows/frogbot-scan-pr.yml

```diff
@@ -0,0 +1,40 @@
+name: "Frogbot Scan Pull Request"
+on:
+  pull_request_target:
+    types: [ opened, synchronize ]
+permissions:
+  pull-requests: write
+  contents: read
+jobs:
+  scan-pull-request:
+    runs-on: ubuntu-latest
+    # A pull request needs to be approved, before Frogbot scans it. Any GitHub user who is associated with the
+    # "frogbot" GitHub environment can approve the pull request to be scanned.
+    environment: frogbot
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          ref: ${{ github.event.pull_request.head.sha }}
+
+      # Install prerequisites
+      - name: Set up Java
+        uses: actions/setup-java@v3
+        with:
+          java-version: "8"
+          distribution: "temurin"
+
+      - uses: jfrog/frogbot@v2
+        env:
+          JFROG_CLI_HOME_DIR: /home/runner/work
```

Comment:
Is it really necessary? I couldn't find it in the template

Reply:
It probably isn't. Removing it. Thanks for noticing this.